ls1intum/Artemis
# Update dependencies

Update dependencies

### Checklist

### Steps for Testing
1. Click through the application and try out different functionalities
2. Everything should work as usual
